File: DotNet_Backend/SafarnamaApplication/SafarnamaApplication/DTOs/CostDTO.cs
namespace SafarnamaApplication.DTOs
{
    public class CostDTO
    {
        public int CostId { get; set; }
        public int? PackageId { get; set; }
        public double? SinglePersonCost { get; set; }
        public double? ExtraPersonCost { get; set; }
        public double? ChildWithBed { get; set; }
        public double? ChildWithoutBed { get; set; }
        public DateOnly? ValidFrom { get; set; }
        public DateOnly? ValidTo { get; set; }
    }
}

File: Java_Backend/E-Tour/src/main/java/dto/CostDTO.java
package dto;

import lombok.*;
import java.time.LocalDate;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class CostDTO {
    private Integer costId;
    private Double singlePersonCost;
    private Double extraPersonCost;
    private Double childWithBed;
    private Double childWithoutBed;
    private LocalDate validFrom;
    private LocalDate validTo;

    
}
